taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.blog.app.css

```diff
@@ -41,18 +41,14 @@ Styleguide Components.DjangoCMS.Blog.App
 
 .app-blog header {
   display: grid;
-  grid-template-areas:
-    'cats'
-    'tags'
-    'attr'
-    'head'
-    'subh';
```

Comment:
**Why?!** Instead of defining the layout of metadata once here, I define it for both `…item.css` and `…page.css`.